packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -815,7 +850,7 @@ describe("handleNewBooking", () => {
               },
             });
 
-            expectWorkflowToBeTriggered();
+            // expectWorkflowToBeTriggered({emails, organizer});
```

Comment:
this fails and is a bug, I created an issue for it: https://github.com/calcom/cal.com/issues/12881